docs/release/release-tasks.md

```diff
@@ -172,6 +165,16 @@ From this point forward changes which should land in the release have to be cher
 3. Update the GitHub Actions to work with the new release version.
    <br>Prior art: [Update actions for 1.5 and make names consistent](https://github.com/kubernetes-sigs/cluster-api/pull/9115)
 
+#### Set a tentative release date for the next minor release
+
+1. Set a tentative release date for the next minor release and document it by creating a `release-X.Y.md` in [docs/release/releases](../../docs/release/releases).
+   <br>Prior art: https://github.com/kubernetes-sigs/cluster-api/pull/9635
+
+#### Assemble next release team
```

Comment:
I think we have to update the issue template to not break links from there to these two headlines

Reply:
Thanks for reminding, updated now